static/app/views/performance/vitalDetail/vitalChart.tsx

```diff
@@ -241,10 +241,6 @@ class VitalChart extends React.Component<Props> {
                       })
                     : [];
 
-                  const seriesMax = getMaxOfSeries(smoothedSeries);
-                  const yAxisMax = Math.max(seriesMax, vitalPoor);
-                  chartOptions.yAxis.max = yAxisMax * 1.1;
```

Comment:
So I removed and let ECharts handle it.

Reply:
Did they have too many decimals? This was added in so you didn't have the timeseries surfing right at the very top of the chart component all the time.

Reply:
@k-fish Yes that was the problem. An alternative we can do is to keep this max value but hide just the max axis label so it's not visible. Would that be preferred?

Reply:
I think perhaps yes, since the max label would be synthetic in this case so it's okay to hide it.

Reply:
Resolved in cbaabdc